content/en/docs/tasks/debug/debug-application/debug-running-pod.md

```diff
@@ -718,3 +718,93 @@ Clean up the Pod when you're finished with it:
 ```shell
 kubectl delete pod myapp
 ```
+
+### Custom Profile {#custom-profile}
+
+{{< feature-state for_k8s_version="v1.31" state="beta" >}}
+
+You can define a partial container spec as a custom profile in either YAML or JSON format, and apply it to an ephemeral container or a debugging container in a copied Pod or a debugging Pod using the `--custom` flag.
+
+{{< note >}}
+Custom profile only supports the modification of the debugging container spec. It does not support the modification of the Pod spec of the debug target.
+Modifications via a custom profile are not allowed for certain fields, including: `command`, `image`, `lifecycle`, `name`, and fields that define access to storage.
```

Comment:
"certain fields" are vague. Please be more specific.

Reply:
I've indicated more specifically in line with the [implementation](https://github.com/kubernetes/kubernetes/blob/0b91def6bcbdde82206ec779791762fb6bc337af/staging/src/k8s.io/kubectl/pkg/cmd/debug/debug.go#L396-L400).